WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -97,4 +164,62 @@ private extension ShippingLabelFormViewModel {
         try? resultsController.performFetch()
         return resultsController.fetchedObjects.first
     }
+
+    func isValidatingAddress(_ validating: Bool, type: ShipType) {
```

Comment:
nit: make more obvious that it's a function to change an internal state, like:

```suggestion
    func updateValidatingAddressState(_ validating: Bool, type: ShipType) {
```

Reply:
👍 ec4fba661839938985e0d23a3ec25bd39cd3447e great suggestion